converged_sound_open_fw_modules_development/FW/src/intel_adsp/build_framework/build_core/custom_firmware_rules.mk

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
This is one huge body of plain makefile rules. We'd need some common make rules how to build modules. Maybe the common rules are done in Cmake and these remain as intel_adsp specific reference?